unit_tests/web_tests/test_web_status.py

```diff
@@ -629,18 +657,35 @@ def test_web_status_vm_overlay_kill(self):
         self.page.click_vm_data_box(self.group_name, self.cloud_name, 'VMs', right_click=True)
         self.assertFalse(self.page.vm_overlay_column_is(3, 'Terminate', 0))
         self.page.click_vm_overlay_close()
+    '''
 
     def test_web_status_vm_overlay_filter_cores(self):
         self.page.wait_until_vms_not_zero(self.group_name, self.cloud_name, 3)
+        print("wait_until_vms_not_zero is good")
```

Comment:
Are these print statements intentional or part of your development? They should be removed if they are no longer needed.